pkg/crc/adminhelper/hosts.go

```diff
@@ -22,13 +23,26 @@ func UpdateHostsFile(instanceIP string, hostnames ...string) error {
 }
 
 func AddToHostsFile(instanceIP string, hostnames ...string) error {
-	return execute(append([]string{"add", instanceIP}, hostnames...)...)
+	var filtered []string
+	for _, hostname := range hostnames {
+		stdout, err := execute("contains", instanceIP, hostname)
```

Comment:
Have you considered using `github.com/code-ready/admin-helper/pkg/hosts` directly? `contains` implementation is:
```
func contains(args []string) error {
        hosts, err := hosts.New()
        if err != nil {
                return err
        }
        if hosts.Contains(args[0], args[1]) {
                fmt.Println("true")
        } else {
                fmt.Println("false")
        }
        return nil
}
```
which is the same amount of code, and would be more efficient.

Reply:
Yes done. I wanted to avoid to vendor admin-helper in crc. We just need to remind to bump it in 2 places.